stackslib/src/chainstate/stacks/index/mod.rs

```diff
@@ -171,6 +171,20 @@ impl From<u32> for MARFValue {
     }
 }
 
+impl From<u64> for MARFValue {
+    fn from(value: u64) -> MARFValue {
+        let h = value.to_le_bytes();
+        let mut d = [0u8; MARF_VALUE_ENCODED_SIZE as usize];
+        if h.len() > MARF_VALUE_ENCODED_SIZE as usize {
+            panic!("Cannot convert a u64 into a MARF Value.");
+        }
+        d.get_mut(..h.len())
+            .expect("Cannot convert a u64 into a MARF Value")
+            .copy_from_slice(&h);
+        MARFValue(d)
+    }
+}
```

Comment:
Are `From<u64> for MARFValue` and `From<MARFValue> for u64` used anymore in this implementation?